Sealer/sealer.cc

```diff
@@ -705,24 +710,29 @@ bool operator<(const FastaRecord& a, const FastaRecord& b)
 void findFlanks(FastaRecord &record,
 	int flanklength,
 	unsigned &gapnumber,
-	map<FastaRecord, map<FastaRecord, Gap> > &flanks)
+	map<FastaRecord, map<FastaRecord, Gap> > &flanks,
+	bool lower)
```

Comment:
Seeing as `lower` is part of `opt` namespace, passing a function parameter here might be adding unnecessary confusion. We can simply use `opt::lower` where needed.